Sources/muterCore/MutationOperators/TokenAwareVisitor.swift

```diff
@@ -4,7 +4,9 @@ class TokenAwareVisitor: SyntaxVisitor, PositionDiscoveringVisitor {
     
     fileprivate(set) var tokensToDiscover = [TokenKind]()
     private(set) var positionsOfToken = [AbsolutePosition]()
-    
+
+    init(_: MuterConfiguration) { }
+
```

Comment:
It would be nice if you declare an empty init too (make the `init(_:MuterConfiguration)` calls the empty init if it is necessary). This way we avoid all the unnecessary changes around the init in this file and in all the Specs that you had to declare a dummy MuterConfiguration.

Reply:
OK, I see 3 options here:
1. Add an empty init for the tests, as you suggest
2. Provide a default value `= MuterConfiguration()` for the parameter
3. Make the parameter Optional with a default value of `nil` (and avoid implicitly constructing an empty configuration object)
Any of those would avoid the changes related to init. Which do you think is preferable? (I lean towards option 3)

Reply:
I would avoid optionals, imho, option 2 makes more sense.
@henrique-morbin-ifood @Uriah-Eisenstein-ATT what do you think?

Reply:
Number 2 is discarded because you cannot add an empty configuration in the production code. It does not make sense, even if you are not using it. It's wrong by the code perspective because we have a Configuration that is not what the user passed.

Number 1 is weird too. I know that it was what I suggested, but I was just trying to avoid the changes.

Number 3, I think it is the best approach here because we let the code clear that it will handle properly in both ways (with or without a configuration).

Reply:
Can we agree on option 3 then?